### DISCUSSION TITLE
go install nuclei failed

### DISCUSSION THREAD

**Discussion by dajima** [projectdiscovery/nuclei#4654] [Q&A] [answered]
#go env
GOVCS=''
GOVERSION='go1.21.6'
GCCGO='gccgo'
GOAMD64='v1'
AR='ar'
CC='gcc'
CXX='g++'
CGO_ENABLED='1'
GOMOD='/dev/null'
GOWORK=''
CGO_CFLAGS='-O2 -g'
CGO_CPPFLAGS=''
CGO_CXXFLAGS='-O2 -g'
CGO_FFLAGS='-O2 -g'
CGO_LDFLAGS='-O2 -g'
PKG_CONFIG='pkg-config'
GOGCCFLAGS='-fPIC -m64 -pthread -Wl,--no-gc-sections -fmessage-length=0 -ffile-prefix-map=/tmp/go-build4105190737=/tmp/go-build -gno-record-gcc-switches'

 go install -v github.com/projectdiscovery/nuclei/v3/cmd/nuclei@latest
runtime/internal/sys
# runtime/internal/sys
go/src/runtime/internal/sys/intrinsics_common.go:9:7: len8tab redeclared in this block
        go/src/runtime/internal/sys/intrinsics.go:76:7: other declaration of len8tab
go/src/runtime/internal/sys/intrinsics_common.go:32:6: Len64 redeclared in this block
        go/src/runtime/internal/sys/intrinsics.go:99:6: other declaration of Len64
go/src/runtime/internal/sys/intrinsics_common.go:50:7: m0 redeclared in this block
        go/src/runtime/internal/sys/intrinsics.go:117:7: other declaration of m0
go/src/runtime/internal/sys/intrinsics_common.go:51:7: m1 redeclared in this block
        go/src/runtime/internal/sys/intrinsics.go:118:7: other declaration of m1
go/src/runtime/internal/sys/intrinsics_common.go:52:7: m2 redeclared in this block
        go/src/runtime/internal/sys/intrinsics.go:119:7: other declaration of m2
go/src/runtime/internal/sys/intrinsics_common.go:55:6: OnesCount64 redeclared in this block
        go/src/runtime/internal/sys/intrinsics.go:122:6: other declaration of OnesCount64
go/src/runtime/internal/sys/intrinsics_common.go:86:6: LeadingZeros64 redeclared in this block
        go/src/runtime/internal/sys/intrinsics.go:153:6: other declaration of LeadingZeros64
go/src/runtime/internal/sys/intrinsics_common.go:89:6: LeadingZeros8 redeclared in this block
        go/src/runtime/internal/sys/intrinsics.go:156:6: other declaration of LeadingZeros8
go/src/runtime/internal/sys/intrinsics_common.go:92:6: Len8 redeclared in this block
        go/src/runtime/internal/sys/intrinsics.go:159:6: other declaration of Len8
go/src/runtime/internal/sys/intrinsics_common.go:101:6: Prefetch redeclared in this block
        go/src/runtime/internal/sys/intrinsics.go:200:6: other declaration of Prefetch
go/src/runtime/internal/sys/intrinsics_common.go:101:6: too many errors
x00358506@X00855612-CiJys:~/bin$ go install -v github.com/projectdiscovery/nuclei/v3/cmd/nuclei@v3.1.5

**Comment by tarunKoyalwar** [comment, marked as answer]
@dajima , looks like a corrupt install of go (ref: https://github.com/golang/go/issues/62004) you need to reinstall go
